utils/handleMessage.js

```diff
@@ -1,5 +1,6 @@
 const request = require("request");
 const logger = require("../config/logger");
+const { sendMessage } =  require("./sendMessage");
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/ParmenidesSartre/Respond-Task1/pullRequest?prid=9934312)
```suggestion
const { sendMessage } = require("./sendMessage");
```